### ISSUE TITLE
Convention proposal for Odds & Evens

### ISSUE THREAD

**Issue by Lel0uch-H** [hanabi/hanabi.github.io#865]
All color clues to be treated normally.

Rank clues are treated the following way:
- Chop focus
  - While loaded
    - 1s to chop is a play clue on the focused card and keep playing right to left unless fixed with a color clue on the next 1 about to play
    - 2s to chop is a play clue on the focused card.
  - While not loaded, chop focus
    - 1s to chop is a save
    - 2s to chop is a save
    - 1s to chop to a playable 1/3/5 calls for an ejection
    - 1s to chop to an unplayable 3 calls for a discharge
    - 2s to chop to an unplayable 4 calls for a charm

- Non chop focus
  - 1s away from chop is a play clue on the left most 1 and keep playing left to right unless fixed as above.
  - 2s away from chop is a play clue on the focused 2.
  - 1s to an unplayable 3 is a discharge. 
  - 1s to an unplayable 5 is an ejection.
  - 2s to an unplayable 4 is a charm.

I am still debating if treating 1-away 3s and 1-away 4s should be different.

**Comment by Zamiell** [comment]
we already played a few games earlier today and we did not play this way.

1's to chop is just a 5 save.
2's to chop is just a 2 save.
always loaded principle applies.

**Comment by Zamiell** [comment]
the stuff about the keep playing thing seems pretty unnecessary to win

**Comment by Zamiell** [comment]
> 1s to an unplayable 3 is a discharge

we didn't do that at all, this should just be a 3 bluff.
i think you are conceptualizing this variant wrong, a 1 clue is not a literal 1 clue its just an odd clue

**Comment by Lel0uch-H** [comment]
> a 1 clue is not a literal 1 clue its just an odd clue

Similar to how the blue clue in the "Extremely ambiguous" variant is an odd clue. It is quite useful to define such cases to account for all the annoying decks that may come our way. Admitted, we also won a 3p game without any of the above stated conventions, but that doesn't mean we will be able to achieve a very high win rate without special conventions

**Comment by Romain672** [comment]
We played a lot of 2p yesterday, and used a lot of color saves (outside when only one slot is available).
2p have less cards saved overall, so having some trash 1s clued from a 5-clue with number was too annoying.

**Comment by mangopie1** [comment]
I agree, dont see a need for extras. Just a variant of existing convention.

So I agree only subset with Lel's list
Odds's to chop is just a 5 save.
Even's to chop is just a 2 save.
always loaded principle applies.

Rank choice ejection for playable Odds/Even for non loaded chop focused clue.


On top of it for "1's"

Odd's non chop focus at starting hand only promised the left most card is a 1. Therefore only left most is playable and the rest just good cards to keep.
